components/smalltalk/squeak5/squeak5.p5m

```diff
@@ -0,0 +1,69 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"), version 1.0.
+# You may only use this file in accordance with the terms of version
+# 1.0 of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source.  A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2020 David Stes
+# (with help from Andreas Wacknitz and Alexander Pyhalov)
+#
```

Comment:
Just remove us helpers here, you are the copyright holder. We just give some tips for integration ;-)